lib/upload.js

```diff
@@ -276,9 +269,10 @@ class Upload {
 
     cb = cb || function () {};
     if (shouldTerminate && this.url != null) {
-      Upload.terminate(this.url, this.options, (err, xhr) => {
+      this._retryAttempt = 0;
```

Comment:
I think setting this._retryAttempt is now unnecessary here because Upload.terminate has it's own retry counter in form of the `retryAttempt` local variable, right?

Reply:
resolved